backend/src/controllers/formContoller.ts

```diff
@@ -41,3 +48,33 @@ export const getForms = async (req: any, res: Response) => {
     res.status(400).json({ error });
   }
 };
+
+export const updateForm = async (req: any, res: Response) => {
```

Comment:
Let's avoid using any on this Request object. Request object. use this resource for inspirations: https://stackoverflow.com/questions/63538665/how-to-type-request-query-in-express-using-typescript